### DISCUSSION TITLE
Broken layout of system related windows (installed version) (Screen turned off)

### DISCUSSION THREAD

**Discussion by cubicYYY** [rustdesk/rustdesk#6551] [General]
## System Info
Laptop: ASUS Redolbook14 with AMD CPU, integrated GPU Vega7, Windows11 23H2
Desktop: ASUS Motherboard + AMD 5800X + NVIDIA RTX 3080, Windows11 23H2
## Problem
When I remote-controlling my desktop, I noticed some problematic layouts of specific windows:
<img width="1280" alt="ed368fb5ea2c175ed4575d9aaa83829a" src="https://github.com/rustdesk/rustdesk/assets/93425234/7705f3f2-08be-471c-a485-0e52376f2722">
**But in another direction (from desktop to laptop), everything just works fine.**
The problem cannot be resolved by refreshing the view.
Also, I noticed those windows that have problems are mostly system-related. Including:
- Windows setting window(overlap, dragging)
- Some installation app
- Start menu
- **The little control panel of Rustdesk itself (upper right corner)**

I can use those windows partly by only using a keyboard(i.e. Tab to switch between buttons). I can see part of the UI components being rendered in the wrong positions but active.
The windows that have problems somewhat not being flushing their cache/buffer? I guess.
**Video is available below to better represent the error.**

### How to Reproduce

Control remote desktop

### Expected Behavior

Display normally like normal windows

### Operating system(s) on local side and remote side

Windows 11 23H2

### RustDesk Version(s) on local side and remote side

1.2.3->1.2.3

### Screenshots

https://github.com/rustdesk/rustdesk/assets/93425234/c3b79658-6000-4384-b075-ac612bf02d30

**Comment by rustdesk** [comment]
controlled side is installed？

**Reply by cubicYYY** [reply]
Both sides are installed. I will try to re-install & restart under Admin during the whole process later and see if it works later.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3017

**Reply by cubicYYY** [reply]
I think the problem is becoming more clear now: if the remote computer's screen is turned off, then this issue will occured.

**Comment by cubicYYY** [comment]
I noticed that even I "unlocked" the setting, next time I switch to the setting tab it still requres. Maybe unlock failed due to trivial privilege settings.

**Comment by cubicYYY** [comment]
After a while(without a reboot or power-off), the issue appeared again :(
Is something about the service or the driver?

**Reply by rustdesk** [reply]
Frankly, we have no clue yet. :(

**Reply by cubicYYY** [reply]
Since the issue occured again after a while and cannot be resolved by a reboot, I will try to figure out whether the GPU driver or Windows defender.
Can I bypass the GPU but using a indirect driver? I installed that but found no corresponding device in my device manager

**Comment by cubicYYY** [comment]
I want to know what the command line option for "rustdesk.exe" `--tray` stands for: I currently set up scheduled tasks to run "rustdesk.exe --service" to start it when logging in to Windows. Is this enough for remote access?

**Reply by bigretromike** [reply]
if you install rustdesk it will run as service in background as soon as windows loadup even if your are not logged in.

**Comment by sevk** [comment]
> Frankly, we have no clue yet. :(

try to  remove the remote monitor power , then use rustdesk connect to the remote .

**Reply by cubicYYY** [reply]
You are on the right way: https://github.com/rustdesk/rustdesk/issues/6682
